docs/stencil-docs/storefront-customization/implementing-WCAG-guidelines.md

```diff
@@ -0,0 +1,47 @@
+
+# Implementing WCAG Guidelines
+
+<div class="otp" id="no-index">
+
+### On this page
+- [Bypass blocks example](#bypass-blocks-example)
+- [Location example](#location-example)
+- [Unusual words example](#unusual-words-example)
+</div>
+
+This article provides Cornerstone code snippets which satisfy current WCAG guidelines or code you can add to meet a WCAG requirement.
+
```

Comment:
For each of these examples, we should have an image of how these things should look when implemented. Otherwise, the developers won't know if they've succeeded.